test/common/TestBed.cpp

```diff
@@ -220,7 +220,7 @@ namespace RcclUnitTesting
     }
   }
 
-  void TestBed::ExecuteCollectives()
+  void TestBed::ExecuteCollectives(std::vector<int> const &currentRanks)
```

Comment:
For consistency with the rest of TestBed calls, we should accept only a single rank, with -1 representing all ranks as before.  This just makes it a loop at the TestBed caller level, instead of building a vector, which is similar in terms of work.